cypress/integration/pages/articles.js

```diff
@@ -15,17 +15,6 @@ const serviceHasCorrectlyRenderedParagraphs = service => service !== 'sinhala';
 const runTests = ({ service }) =>
   describe(`Tests`, () => {
     describe(`Metadata`, () => {
-      it('should have lang and dir attributes', () => {
```

Comment:
Since you've moved the html lang check to the shared file, you should delete the duplicated tests here too: 
https://github.com/bbc/simorgh/blob/5a10c31a5f6cb64e7f425a4eac4bb68feb1715eb/cypress/integration/pages/articles.js#L86

Reply:
Thanks I missed this one